x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -106,6 +107,7 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
   const { addWarning } = useAppToasts();
   // TODO: Once we are past experimental phase this code should be removed
   const ruleRegistryEnabled = useIsExperimentalFeatureEnabled('ruleRegistryEnabled');
+  const ACTION_BUTTON_COUNT = 4;
```

Comment:
💅 Same here, maybe this can be moved to outside the component

Reply:
I don't know if it has use anywhere else at the moment so it may just be pre-optimization 🤷🏾‍♂️ . It might be worth just making it a constant within the component itself like `const actionButtonCount = 4;` so we retain context without feeling like it should live somewhere else?